### PR TITLE
OCPBUGS-1979: update etcd build version to 1.16

### PR DESCRIPTION
**Pull request by tjungblu**
required for https://github.com/openshift/etcd/pull/148

### CODE REVIEW

File: ci-operator/config/openshift/etcd/openshift-etcd-openshift-4.8.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: rhel-8-release-openshift-4.8
+    tag: rhel-8-release-openshift-4.3
```
Comment (Elbehery) [inline ci-operator/config/openshift/etcd/openshift-etcd-openshift-4.8.yaml:15]: I dont understand, why we need 4.3 for ocp 4.8 ?

Comment (tjungblu) [reply ci-operator/config/openshift/etcd/openshift-etcd-openshift-4.8.yaml:15]: neither do I, but the 4.8 tag does not exist

**Comment by EmilyM1** [comment]
Does this need changes to vendor packages?

**Comment by hasbro17** [comment]
Guess we're not in the approvers list for https://github.com/openshift/release/blob/master/ci-operator/config/openshift/etcd/OWNERS
We'll need an update and backport of that.

@deads2k If you could approve in the meantime that would be great.

**Comment by tjungblu** [comment]
/retitle OCPBUGS-1979: update etcd build version to 1.16

**Pull request merged**